docs/plain-bundle-spec.md

```diff
@@ -161,3 +161,46 @@ spec:
   provisionerClassName: core.rukpak.io/plain
 EOF
 ```
+
+## Private image registries
+
+A Bundle can reference content in a private image registry by creating an imagePullSecret in the namespace that the provisioner is deployed.
```

Comment:
nit: Is the intent here to reference the exact key that is used in the spec?
```suggestion
A Bundle can reference content in a private image registry by creating an `pullSecret` in the namespace that the provisioner is deployed.
```

Reply:
Thanks! I change it.